### ISSUE TITLE
UI Bug for Assessments with many columns

### ISSUE THREAD

**Issue by nayak16** [autolab/Autolab#685]
<img width="1428" alt="screen shot 2016-09-10 at 1 35 16 pm" src="https://cloud.githubusercontent.com/assets/6308671/18412403/8a9ca58c-775b-11e6-9dd0-fdda337f872a.png">

**Comment by nayak16** [comment]
Should have sticky navbar and scroll the bottom section

**Comment by zyx-billy** [comment]
The same problem also happens with the gradesheet, and the assessment main page (where you submit files).
![gradesheet_too_long](https://cloud.githubusercontent.com/assets/3676913/18444843/dd42f50a-78e8-11e6-95dc-e8fa1c60d330.png)
I'll get on it.

**Comment by zyx-billy** [comment]
Adding a wrapper and setting overflow-x produces this:
![table_overflow_auto](https://cloud.githubusercontent.com/assets/3676913/18530954/cc8da600-7aa1-11e6-9abd-65ab4eeed468.png)
Looks kind of weird?

**Comment by devanshk** [comment]
Yeah, it's not ideal. Could be good for a hotfix until 2.0?

**Comment by zyx-billy** [comment]
Yeah at most a temporary fix. Perhaps we don't even need to make a hotfix for this. Seems not that big of a deal functionality wise... (and honestly it's easier to see all the grades without the fix)

**Comment by zyx-billy** [comment]
By the way, why are we manually setting pagebody width to be 970px even on screens wider than 1200px?  (The bootstrap default is 1170px) It makes pages like this only use a small part of the page in the center.

**Comment by 20wildmanj** [comment]
Resolved by #1609
